react/src/components/PayButton/PayButton.tsx

```diff
@@ -105,6 +107,11 @@ PayButton.defaultProps = {
   randomSatoshis: true,
   successText: 'Thank you!',
   disabled: false,
+<<<<<<< HEAD
```

Comment:
This looks like it got messed up when reversing it arounnd?

Reply:
Yup, Windows is undoing my changes in VSCode for some reason.